functions-and-scope.js

```diff
@@ -72,6 +106,23 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 
 // ---- Verwachte uitkomst: 9
 
+// For loop
+// If it's higher than the last highest value we saved.
+// Let saved before the loop
+
+function highestGrade(grades) {
+    let highestGrade = 0;
```

Comment:
Voor deze opdracht niet heel erg, maar wees je ervan bewust dat als je een array als `[-3, -1, -2, -5]` meegeeft aan deze functie, dat je dan `0` terugkrijgt. Je kan daarom altijd beter de eerste waarde van de array pakken
```suggestion
    let highestGrade = grades[0];
```